docker/android/build.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -e -x
+
+export FASTLANE_DISABLE_COLORS=1
+export REALM_DISABLE_ANALYTICS=1
+export STATUS_RELEASE_STORE_PASSWORD=1TteVCyHykCa
```

Comment:
What credentials are these?

Reply:
Ah, that's from some of my tests. Let me drop that.